docs/en/rules/Azure.RSV.ReplicationAlert.md

```diff
@@ -70,6 +72,7 @@ resource testRecoveryServices 'Microsoft.RecoveryServices/vaults/replicationAler
     customEmailAddresses: [
       'example@email.com'
     ]
+    locale: 'en-US'
```

Comment:
Let's add this `locale` property to the ARM Template code as well so that it is the same between both.

Reply:
Thanks for adding @BernieWhite